src/visions/types/time.py

```diff
@@ -9,12 +9,12 @@
 
 
 def test_time(series):
-    dtseries = series.dropna().dt.date
+    dtseries = series.dt.date
```

Comment:
Do you know why we were dropping na here? There must have been some reason right (I'm sure this was me but nothing comes to mind so maybe not?).

Reply:
Actually the dropna is required for the test. Just forgot to add the nullable decorator since this function is not in use.